internal/graph/loaders/sinks.go

```diff
@@ -0,0 +1,13 @@
+package loaders
+
+type Sinks struct {
+	ComponentBySink *ComponentLoader
+	PodsBySink      *PodSliceLoader
+	TopicBySink     *TopicLoader
+}
+
+func configureSinks(loaders *Loaders) {
+	loader := &Sinks{}
+	loaders.SinkLoader = loader
```

Comment:
could this just be `loaders.SinkLoader = &Sinks{}`

Reply:
this is not complete. It will configure the loaders on the type like the others do